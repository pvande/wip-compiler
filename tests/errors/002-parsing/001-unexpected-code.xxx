x := 1 something something something something
