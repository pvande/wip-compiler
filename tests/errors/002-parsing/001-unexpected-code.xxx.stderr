Error: "Unexpected code following statement"
In [1;37mtests/errors/002-parsing/001-unexpected-code.xxx[0m on line [1;37m1[0m

> [0;36mx := 1 something something something something[0m
  [0;31m       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
